docs/integrators/deep-history-squad.md

```diff
@@ -102,10 +102,13 @@ Since each observer of a deep-history squad must have a non-pruned history, thei
 
 [comment]: # (mx-context-auto)
 
-## Observer configuration
-
-A deep history squad has its observers set up to retain the whole, non-pruned history of the blockchain. That is, the observers must be started with the flag `--operation-mode=historical-balances`.
+## Observer installation and configuration
 
+The installation of a deep history squad is the same as that of a [regular observing squad](/integrators/observing-squad).
+The difference is that the deep history observers are set up to retain the whole, non-pruned history of the blockchain. Therefore, the following flag must be added in the `config/variables.cfg` file before installation:
```

Comment:
`before installation` - maybe `before running the _install squad_ command of the installation scripts`? (or something similar).

Reply:
Done.